resource-server/src/main/java/com/inhabas/api/web/MyInfoController.java

```diff
@@ -225,6 +230,93 @@ public ResponseEntity<Void> handleMyInfoRequest(
     return ResponseEntity.noContent().build();
   }
 
-  // [모임, 글, 댓글, 예산신청 조회] 추후 개발 예정
+  @Operation(summary = "내가 쓴 게시글 목록 조회")
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "200",
+            content = {@Content(schema = @Schema(implementation = MyBoardDto.class))}),
+      })
+  @GetMapping("/myInfo/boards")
+  public ResponseEntity<PagedResponseDto<MyBoardDto>> getBoardList(
+      @Parameter(description = "페이지", example = "0")
+          @RequestParam(name = "page", defaultValue = "0")
+          int page,
+      @Parameter(description = "페이지당 개수", example = "10")
+          @RequestParam(name = "size", defaultValue = "10")
+          int size,
+      @Authenticated Long memberId) {
+
+    Pageable pageable = PageRequest.of(page, size);
+    List<MyBoardDto> allDtoList = myInfoService.getMyBoards();
+    List<MyBoardDto> pagedDtoList = PageUtil.getPagedDtoList(pageable, allDtoList);
```

Comment:
memberId를 넘겨서 myInfoService.getMyBoards에서 memberRepository.findById로 호출하는게 나아보입니다. 이미 확인한 securitycontext를 service에서 또 확인할 필요는 없지 않을까요?

Reply:
```
@Service
@Slf4j
@RequiredArgsConstructor
public class MyInfoServiceImpl implements MyInfoService {
...
 @Override
  @Transactional(readOnly = true)
  public List<MyBoardDto> getMyBoards(memberId) {

    List<MyBoardDto> myBoardDtoList = new ArrayList<>();
    myBoardDtoList.addAll(myInfoRepository.findAllBoardsByMemberId(memberId));
    if (SecurityContextHolder.getContext() == null) {
      throw new InvalidAuthorityException();
    }

    return myBoardDtoList;
  }
...
  }
```
이렇게 서비스 구현부분  코드 수정하면 될까요? service부분에서  securitycontext 확인하는 부분 제거하고 controller에서 memberId를 매개변수로 받을 수 있게 코드 수정했습니다.

Reply:
`` 결과적으로 컨트롤러는 이렇게 바뀌는게 맞죠?
```suggestion
      @Authenticated Long memberId) {

    Pageable pageable = PageRequest.of(page, size);
    List<MyBoardDto> allDtoList = myInfoService.getMyBoards(memberId);
    List<MyBoardDto> pagedDtoList = PageUtil.getPagedDtoList(pageable, allDtoList);
```

Reply:
@whitem4rk 시간 나실 때 확인 부탁드립니다!

Reply:
네 맞습니다.